src/templates/PoolV2/NewPoolOperations/Form/ListOfAllAsset/styles.ts

```diff
@@ -62,35 +62,19 @@ export const tokenLogo = styled.span`
   }
 `
 
-export const Input = styled.input`
+export const value = styled.p`
```

Comment:
It's a component, so it should be named using Pascal Case

```suggestion
export const Value = styled.p`
```